src/main/java/com/iota/iri/service/API.java

```diff
@@ -657,6 +664,32 @@ private static void incEllapsedTimeGetTxToApprove(long ellapsedTime) {
         ellapsedTime_getTxToApprove += ellapsedTime;
     }
 
+    /**
+     * <p>
+     *     Get the confirmation confidences for a set of transactions.
+     *     This is for determining if a transaction was accepted and confirmed by the network or not.
+     * </p>
+     * <p>
+     *     This API call returns a list of floating point values in the [0,1] interval, 
+     *     in the same order as the submitted transactions.<br/>
+     *     The higher the confidence for a given transaction, the more likely it is to be accepted.
+     * </p>
+     * Returns an {@link com.iota.iri.service.dto.ErrorResponse} if a tip is missing or the subtangle is not solid
```

Comment:
I think this is a leftover c/p? what tip/non-solid?

Reply:
yeah, and remove the "not solid"  part?

Reply:
👍